--- ffmpeg-build-static-binaries/include/ffnvcodec/dynlink_cuda.h.orig	2018-10-14 21:53:25.000000000 +0300
+++ ffmpeg-build-static-binaries/include/ffnvcodec/dynlink_cuda.h	2018-10-14 22:01:15.791295749 +0300
@@ -100,9 +100,12 @@
     CU_EXTERNAL_MEMORY_HANDLE_TYPE_D3D12_RESOURCE   = 5,
 } CUexternalMemoryHandleType;
 
+#ifndef CU_UUID_HAS_BEEN_DEFINED
+#define CU_UUID_HAS_BEEN_DEFINED
 typedef struct CUuuid_st {
     char bytes[16];
 } CUuuid;
+#endif
 
 typedef struct CUDA_MEMCPY2D_st {
     size_t srcXInBytes;
